doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -727,6 +727,30 @@ declare your constant as rigid for proof search using the command
 Strategies for rewriting
 ------------------------
 
+Usage
+~~~~~
+
+.. tacn:: rewrite_strat @rewstrategy {? in @ident }
+   :name: rewrite_strat
+
+   Rewrite using the strategy s in hypothesis ident or the conclusion.
```

Comment:
what is `s` here?

Reply:
```suggestion
   Rewrite using the strategy `@rewstrategy` in hypothesis `@ident` or the conclusion.
```

(maybe you need some sort of :something: before these?)

Reply:
Yes, `:n:`

Reply:
> what is `s` here?

Dunno, I only moved the text.  I will fix though.